crates/sui-rpc-api/src/client/mod.rs

```diff
@@ -50,7 +50,11 @@ impl Client {
         let mut endpoint = tonic::transport::Endpoint::from(uri.clone());
         if uri.scheme() == Some(&http::uri::Scheme::HTTPS) {
             endpoint = endpoint
-                .tls_config(ClientTlsConfig::new().with_enabled_roots())
+                .tls_config(
+                    ClientTlsConfig::new()
+                        .with_enabled_roots()
+                        .assume_http2(true),
```

Comment:
Is this necessary?

Reply:
this was needed previously when connecting to a private full node via a basic auth proxy, ALPN negotiation wasn't working. But now we are connecting to the full node directly inside the VPC so it's not needed anymore, I will revert this change.